src/main/java/com/autotune/database/dao/ExperimentDAOImpl.java

```diff
@@ -75,6 +78,47 @@ public ValidationOutputData addExperimentToDB(KruizeExperimentEntry kruizeExperi
         return validationOutputData;
     }
 
+    @Override
+    public void deletePartitions(int daysCount) {
+        LOGGER.info("Threshold is set to {}", daysCount);
+        // Calculate the date 'daysCount' days ago
+        Calendar calendar = Calendar.getInstance();
+        calendar.add(Calendar.DAY_OF_YEAR, -daysCount);
+        Date cutoffDate = calendar.getTime();
+
+        // Format the date to match the table naming pattern
+        SimpleDateFormat dateFormat = new SimpleDateFormat("yyyyMMdd");
+        String formattedDate = dateFormat.format(cutoffDate);
+
+        try (Session session = KruizeHibernateUtil.getSessionFactory().openSession()) {
+            List<String> tablenames = session.createNativeQuery(SELECT_ALL_KRUIZE_TABLES).getResultList();
+            for (String tableName : tablenames) {
+                String datePart = null;
+                if (tableName.startsWith("kruize_results_")) {
+                    datePart = tableName.substring("kruize_results_".length());
+                } else if (tableName.startsWith("kruize_recommendations_")) {
+                    datePart = tableName.substring("kruize_recommendations_".length());
+                }
+                if (null != datePart) {
+                    Date tableDate = new SimpleDateFormat("yyyyMMdd").parse(datePart);
+                    // Compare the date part with the cutoff date
+                    if (tableDate.after(cutoffDate)) {
+                        LOGGER.info("Table not eligible for deletion: " + tableName);
```

Comment:
Should we set it as `debug` ?

Reply:
It's a less log, and I want to confirm whether the correct table is being deleted.